src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Api/Experimental/IAgentExperimental.cs

```diff
@@ -17,5 +19,15 @@ public interface IAgentExperimental
         void RecordSupportabilityMetric(string metricName, int count = 1);
 
         void IncrementLogLinesCount(string logLevel);
```

Comment:
I think this method should be removed, and `RecordLogMessage` should call the `AgentHealthReporter` to increment any counts. I can see it being error prone to have separate calls for these metrics (unless there is a use case I'm not thinking of).

Reply:
Yeah, now that we attempt to record everything, it makes sense to move it.